### ISSUE TITLE
VerticalTabs: Invalid aria attributes

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise#5712]
**Describe the bug**
this is an issue raised by VPAT
The tabs design pattern used in the side navigation is not implemented correctly. 
* attribute aria-multiselectable is not valid on an element with role="tablist"
* elements with role="tab" should be immediate children of elements with role="tablist"
* attribute aria-expanded is not valid on an element with role="tab"

**To Reproduce**
1. Go to http://localhost:4200/ids-enterprise-ng-demo/tabs-vertical or https://main-enterprise.demo.design.infor.com/components/tabs-vertical/example-index.html

**Version**
- ids-enterprise-ng: 10.7.0

**Screenshots**
![image](https://user-images.githubusercontent.com/29673378/136193333-5a8be1fe-c59e-4bc4-8f3b-e9f755c088a5.png)

**Platform**
 - Device (if applicable) :NA
 - OS Version: Windows 10
 - Browser Name: Firefox
 - Browser Version: 92.0.1

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-37695

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
